api/spec/v1.yaml

```diff
@@ -1235,6 +1232,14 @@ components:
       pattern: '^oasis1[a-z0-9]{40}$'
       example: "oasis1qpg2xuz46g53737343r20yxeddhlvc2ldqsjh70p"
       description: An Oasis-style (bech32) address.
+    EthOrOasisAddress:
+      type: string
+      pattern: '^oasis1[a-z0-9]{40}$|^(0x)?[0-9a-fA-F]{40}$'
+      examples:
+        eth: 
+          value: "0x0cAf4748697AE0751728824527044D8c583342c6"
+        oasis:
+          value: "oasis1qpg2xuz46g53737343r20yxeddhlvc2ldqsjh70p"
```

Comment:
TIL about `examples` vs `example`. Nice.

Reply:
our openapi linter doesn't like it 💀

Reply:
You nerd-sniped me. That took some careful spec reading :/

The [Parameter](https://swagger.io/specification/#parameter-object) object takes `examples`, but [Schema](https://swagger.io/specification/) only takes `example`, and notes that "The example property has been deprecated in favor of the JSON Schema examples keyword."

From what I can tell, "the JSON Schema examples keyword" is simply the one at Parameter level.

So this works:
```yaml
        - in: query
          name: rel
          schema:
            allOf: [$ref: '#/components/schemas/EthOrOasisAddress']
          examples:
            eth: 
              value: "0x0cAf4748697AE0751728824527044D8c583342c6"
            oasis:
              value: "oasis1qpg2xuz46g53737343r20yxeddhlvc2ldqsjh70p"
```

Unfortunately that's pretty verbose and repetitive: Every time we specify the `EthOrOasisAddress` schema, we also have to specify the examples. I shortened it a little more by reusing example objects, and pushed a commit because it's easier than explaining. Feel free to tweak further.

Here's how it renders: 
![image](https://github.com/oasisprotocol/nexus/assets/629970/8dca94bc-3cce-4615-944e-aef16b1d434a)

Reply:
Thank you for delving into that rabbit hole and adding the fix! Looks great; merging asap